eth/vm/computation.py

```diff
@@ -150,6 +229,17 @@ def __init__(self,
         code = message.code
         self.code = CodeStream(code)
 
+        # Stack manipulation is performance-sensitive code
+        # Avoid method call overhead by assigning stack method directly to object
+        self.stack_push_int = self._stack.push_int  # type: ignore # method assignment unsupported
```

Comment:
Can you make these `cached_property` instaed so that it's lazy and fast.  That potentially bypasses the `type: ignore` as well.

Reply:
Interesting, so a cached property that is an underlying method. Sure, I don't see any problems :+1: .

Reply:
Yup, I'm happy with this. It's still a tiny bit surprising, but much less than assigning the method in init.